src/main/scala/nl/biopet/utils/process/AsyncProcess.scala

```diff
@@ -100,21 +100,38 @@ trait Sys {
 
   var maxRunningProcesses: Int = 5
 
-  def exec(cmd: String): ExecResult = exec(cmd.split(" "))
+  def exec(cmd: String,
+           cwd: Option[File],
+           env: Map[String, String]): ExecResult =
+    exec(cmd.split(" "), cwd, env)
+
+  def exec(cmd: String): ExecResult =
+    exec(cmd.split(" "))
+
+  def exec(cmd: String, cwd: Option[File]): ExecResult =
+    exec(cmd.split(" "), cwd)
+
+  def exec(cmd: String, env: Map[String, String]): ExecResult =
```

Comment:
Can we combine this 3 method into this?
```scala
def exec(cmd: String,
         cwd: Option[File] = None,
         env: Map[String, String] = Map()): ExecResult = {
  exec(cmd.split(" "), cwd, env)
}
```

It might complain about a duplicate overloading method, then this method should be changed to `exeString`